packages/animations/lib/src/open_container.dart

```diff
@@ -389,6 +389,22 @@ class _OpenContainerRoute extends ModalRoute<void> {
       weight: 8 / 12,
     ),
   ]);
+  final TweenSequence<Color> _scrimFadeInTween = TweenSequence<Color>(
+    <TweenSequenceItem<Color>>[
+      TweenSequenceItem<Color>(
+        tween: ColorTween(begin: Colors.transparent, end: Colors.black54),
+        weight: 4 / 12,
```

Comment:
Is there a reason we use 4 / 12 and 8 / 12 instead of 1 / 3 and 2 / 3 for the weights?

Reply:
Also, looking at the scrim spec, I think the weights are supposed to be 3 / 10 and 7 / 10 here

Reply:
Talked offline. This is the fade-through variant, which has this slightly different.